docs/filters.md

```diff
@@ -148,7 +148,7 @@ Matches all issues with a body (or description) containing the given string.
 
 *Expects a string*
 
-Matches all issues associated with any milestones whose names contain the given string.
+Matches all issues associated with any milestones whose names contain the given string. `current` or `curr` can be used to refer to an open milestone with earliest due date, or an overdue open milestone with open issues if any. `prev`, `prev-1`, `prev-2`, `next`, `next+1, `next+2`, etc. can be used to refer to milestones before or after `current`, sorted by due date.
 
```

Comment:
For documentation purpose? I think it may make it confusing to user, like
am I supposed to type `current` or `$current`.

If you are thinking of using it in the filter, I think it's a good idea in
case user has a milestone named `V0.1 previously` and he wants to search
for that `prev` instead of previous milestone.
On Oct 12, 2015 9:11 PM, "Damith C. Rajapakse" <notifications@github.com>
wrote:

> In docs/filters.md
> <https://github.com/HubTurbo/HubTurbo/pull/1050#discussion_r41754198>:
>
> > @@ -148,7 +148,7 @@ Matches all issues with a body (or description) containing the given string.
> >
> >  *Expects a string*
> >
> > -Matches all issues associated with any milestones whose names contain the given string.
> > +Matches all issues associated with any milestones whose names contain the given string. `current` or `curr` can be used to refer to an open milestone with earliest due date, or an overdue open milestone with open issues if any. `prev`, `prev-1`, `prev-2`, `next`, `next+1, `next+2`, etc. can be used to refer to milestones before or after `current`, sorted by due date.
> >
>
> Should we use a special symbol to indicate variables? e.g. $current?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HubTurbo/HubTurbo/pull/1050/files#r41754198>.
>